resultstore_client.py

```diff
@@ -436,3 +580,21 @@ def __file_upload_helper(self, target_id, action_id, config_id):
             timing=common_pb2.Timing(duration=duration))
         self.update_target(new_target,
                            ['timing.duration', 'status_attributes'])
+
+    def __upload_files(self, target_id):
```

Comment:
This is more of an internal Google thing, but use single _ instead of __ for private methods and variables.

Reply:
+1